spec/bolt/cli_spec.rb

```diff
@@ -728,7 +741,7 @@ def stub_config(config, file_content = nil)
             task_options: task_params
           }
           expect { cli.execute(options) }.to raise_error(
-            Bolt::CLIError, /Task not found: dne::task1/
+            Bolt::CLIError, /^Task not found: dne::task1/
```

Comment:
Is there something you think might wrap this message that wasn't intentional?

Reply:
Ah, that may be unnecessary since it's just checking the exception.